Task_2_2_1/src/main/java/r/nsu/fit/masud/Main.java

```diff
@@ -0,0 +1,70 @@
+package r.nsu.fit.masud;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        // Read JSON input file
+        JsonParser parser = new JsonParser();
+        try (FileReader reader = new FileReader("C:\\Users\\Asus\\IdeaProjects\\oop\\Task_2_2_1\\MeinePizza.json")) {
+            JsonElement root = parser.parse(reader);
+            JsonArray ordersJsonArray = root.getAsJsonArray();
+
+            // Convert JSON to objects
+            Gson gson = new GsonBuilder().create();
```

Comment:
You should load from JSON the configuration of pizzeria (baker and courier paramters and count, warehouse capacity, etc.), not orders

For orders the best option is to create Clients, which will generate orders on the fly in separate threads.